README.md

```diff
@@ -0,0 +1,13 @@
+# TensorChord Internals
+
+## Dev
+
+Add new Markdown files to `/content/docs` folder.
+
+For the 1st time, you will need to get the theme from git submodule. Run `make update-theme` to sync the theme repo.
+
+Run `make dev` to create a service to host the site locally.
+
+## License
+
+This repo is under the [Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) license.
```

Comment:
Does this license work for you? @gaocegege @VoVAllen

Reply:
LGTM